recipes/pypints/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
The license is in the repo, I guess you do not need to ship it with the recipe.

Reply:
The license file is now removed